webSockets/payloadParser.go

```diff
@@ -91,12 +93,49 @@ func (wpp *websocketPayloadParser) ExtractPayloadData(payload []byte) (*data.Pay
 	return payloadData, nil
 }
 
+// ExtendPayloadWithCounter will put in the provided payload the provided counter if needed
+func (wpc *webSocketsPayloadConverter) ExtendPayloadWithCounter(payload []byte, counter uint64, withAcknowledge bool) []byte {
```

Comment:
please merge ExtendPayloadWithCounter and ExtendPayloadWithOperationType into a single function and name it to something like `ConstructPayloadData` to match the `ExtractPayloadData`

Reply:
done